custom_components/unifiprotect/services.py

```diff
@@ -22,65 +24,92 @@ def _async_all_ufp_instances(hass: HomeAssistant) -> list[ProtectApiClient]:
     ]
 
 
+def _async_unifi_mac_from_hass(mac: str):
+    # MAC addresses in UFP are always caps
+    return mac.replace(":", "").upper()
+
+
+def _async_get_macs_for_device(device_entry: dr.DeviceEntry) -> list[str]:
```

Comment:
```suggestion
@callback
def _async_get_macs_for_device(device_entry: dr.DeviceEntry) -> list[str]:
```

I don't think we ever pass these to `hass.add_job` but if we did they would run in the executor without the decortator